src/test/steps/signUpSteps.ts

```diff
@@ -0,0 +1,20 @@
+import { Given, When, Then } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import basePage from '../../pages/basePage';
+
+Given('User passes the authorization', async function () {
+  await basePage.authPage.authorizeUser();
+});
```

Comment:
We already have this step defined: https://github.com/YahyoMahmudov/E2E-Testing/blob/8c1d6e21a34f940e096be74e05c7af45b48e9b11/src/test/steps/loginSteps.ts#L6